scripts/update/10-dependencies.sh

```diff
@@ -17,3 +17,19 @@ if [[ ${missing[0]} != "" ]]; then
 else
     echo_log_only "No dependencies required to install"
 fi
+
+if [[ $(_os_distro) == "Debian" ]]; then
+    if ! grep -q contrib /etc/apt/sources.list; then
+        echo_info "Enabling contrib repo"
+        apt-add-repository contrib >> ${log} 2>&1
+        trigger_apt_update=true
+    fi
+    if ! grep -q non-free /etc/apt/sources.list; then
+        echo_info "Enabling non-free repo"
+        apt-add-repository non-free >> ${log} 2>&1
+        trigger_apt_update=true
+    fi
+    if [[ $trigger_apt_update == "true" ]]; then
+        apt_update
+    fi
+fi
```

Comment:
Should this be above the actual dependency installation?

Reply:
`apt-add-repository` will not be present on most Debian installs, thus it needs to be done after the deps are checked. I don't think any of the global deps themselves should be coming from non-free or contrib

Reply:
👍 thanks